EIPS/eip-7610.md

```diff
@@ -0,0 +1,99 @@
+---
+eip: 7610
```

Comment:
```suggestion
eip: 7670
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.